library/Core/src/Juvix/Core/Common/Context.hs

```diff
@@ -37,6 +37,7 @@ nameSymbolFromSymbol = NameSymbol.fromSymbol
 -- Body
 --------------------------------------------------------------------------------
 
+-- |
```

Comment:
hehe is this on purpose?

Reply:
Not particularly, I was going to comment it for the website but got sidetracked and didn't have a custom branch setup for it.

Reply:
Found it funny because the function name is `empty`